conjur/login/credential_data.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""
+CredentialData module
+
+This module represents an object that holds netrc data
+"""
+
+# pylint: disable=too-few-public-methods
+class CredentialData:
+    """
+    Used for setting user input data to login to Conjur
+    """
+
+    def __init__(self, machine=None, login=None, password=None):
+        self.machine = machine
+        self.login = login
+
+        # Purposefully not named the same for clarity in code. .netrc file format standard
+        # uses password but this value is actually the user/host api key. This convention should
+        # be kept. See https://www.labkey.org/Documentation/wiki-page.view?name=netrc especially
+        # the 'Use API Keys' section
```

Comment:
this clarity isn't clear to me now... 😉 
Does it store password or API key?

Reply:
Stores the API key _but_ the key is 'password' in the netrc

Reply:
I updated the class attribute to be api_key